packages/cards/stories/TemplateCard.stories.tsx

```diff
@@ -17,7 +17,7 @@ export default {
     title: 'Demo Survey (Satisfaction Ice Cream)',
     imageSrc:
       'https://images.unsplash.com/photo-1534551767192-78b8dd45b51b?fit=crop&w=220&h=202&q=60',
-    views: '213',
+    views: 213,
```

Comment:
Do the views actually come back as numbers from GraphQL?
If so, we should also directly adjust the type definition in https://github.com/easyfeedback/RCL/blob/185bd0e8d0b4f3c0e36567d86d00dd3d4a22817d/packages/cards/src/TemplateCard.tsx#L25

Reply:
see last commit! :dancers: